src/bin/nydusd/fs_cache.rs

```diff
@@ -743,6 +747,166 @@ impl FsCacheHandler {
         unsafe { fscache_cread(fd as i32, hdr.msg_id as u64).unwrap() };
     }
 
+    pub fn cull_cache(&self, blob_id: String) -> Result<()> {
+        //should hold lock during cull_cache, cause we need change pwd when call inuse and cull
+        let children = fs::read_dir(self.cache_dir.lock().unwrap().clone())?;
```

Comment:
current working dir is a pre-process resource,  so do we need to launch a new process to do cache cull ?

Reply:
That's a little heavy:( @bergwolf what's your thoughts here?
BTW, why does fscache depend on CWD?

Reply:
The cachefiles kernel APIs depend on CWD ,  inuse and cull cmd only accept file name, not a path. Before call inuse or cull the user daemon should chdir to the dir of blob file.  Maybe we can make these cachefiles kernel API more friendly in the future~

Reply:
we can not use chdir()  in a multi-threaded or concurrent application, in case of some race conditions. Actually this func will only called by api-server , which is  a single-thread server, and no other func will call chdir(), so I think it is safe, also we do not need a lock. Does this make sense?

Reply:
Let's go that way first. If it turns out to be problem, we can fork a new process to do the cull.